Add missing ROM folders and two IDE CD-ROM drives

unreal9010 opened the pull request:
This commit adds the missing ROM folders and the following IDE CD-ROM drives:

- Kenwood True-X UCR-421 (72X) - allegedly the world's fastest CD-ROM drive from the late 1999;
   based on data from https://www.youtube.com/watch?v=QNY4DFhf6xM and 
   https://www.targetpc.com/hardware/cd-rom/kenwood72x/index2.shtml
   
- NEC CDR-1300A (6X)

The max number of CD drive models is now set to 12. Few minor corrections related to identification strings of Creative/Matshita drives. 
Will start gathering data to add the Quantum Fireball series of IDE hard drives.

leilei- commented:
IMO Kenwood emulation would require more than just the ID.  It's got a lot of failure edge cases associated with the multi laser tech

unreal9010 commented:
Thank you for letting me know.
Also, is there any website where I could check ATAPI/Windows id strings for every drive I added? I found some documentation online, but it doesn't cover all the drives I added.
